### ISSUE TITLE
Downloads are incorrectly named.

### ISSUE THREAD

**Issue by kurtman1111** [mikew/ss-plex.bundle#46]
I searched for the tv series Continuum but only Continuum Web Series is found so I added it to my favorites and downloaded the first 5 shows. These are pulling metadata for Continuum Web Series but the episodes are actually the tv series Continuum. I can go into the downloaded files one at a time and rename them by dropping "Web Series" from the name then they will rematch with the correct medadata. Now the correct information shows in TV Shows but I still have a mismatch in my ss-plex Favorites folder and have to correct the file name for every download. Is there a way for me to fix this mismatch in my Favorites folder? Sorry for asking for help on this, Mike, I can see that you are being bombed with a lot of questions.

**Comment by karmadelic702** [comment]
I've noticed this same issue with Continuum a while ago. Wasn't bothering me too much. Great show by the way! Were you able to look at the sources to see if they are incorrectly sending metadata to SSPlex? I wonder if it is possible if one of Plex's metadata agents is the culprit. Have you tried disabling the metadata agents one at time to see if one of them is causing the issue?

**Comment by kurtman1111** [comment]
Yes, I have tried this and it doesn't help. At this point I have downloaded all of these shows from an alternate sourse, named them properly and put them on my TV Shows folder and Plex downloads the correct metadata. For some reason when I search for Continuum only Continuum Web Series is available to choose and it downloads the shows for Continuum but names the files Continuum Web Series. I have also found that a search within ss-plex cannot find "Eleventh Hour US" or "Humans". I'm guessing that this is a data source problem beyond Mike's or Github's control but they won't even bother to respond to advise. Anyway, even though there is poor to no service, ss-plex is still the best plugin that exists. Thanks for your thought.
